VAMobile/src/screens/BenefitsScreen/BenefitsScreen.tsx

```diff
@@ -86,7 +86,13 @@ function BenefitsScreen({}: BenefitsScreenProps) {
         />
         <LargeNavButton title={t('lettersAndDocs.title')} onPress={onLetters} />
         <LargeNavButton title={t('disabilityRating.title')} onPress={onDisabilityRatings} />
-        {showAlert && <CategoryLandingAlert text={alertMessage} isError={claimsAndAppealsError || !!nonFatalErrors} />}
+        {showAlert && (
+          <CategoryLandingAlert
+            text={alertMessage}
+            isError={claimsAndAppealsError || !!nonFatalErrors}
+            bypassHaptics={false}
```

Comment:
dropping the bypass should eliminate the need for any of these changes.

Reply:
reverted!